content/docs/user-guide/contributing/docs.md

```diff
@@ -193,9 +203,9 @@ We try to use a casual and fun tone in our docs. We also avoid authoritative
 language such as "As you can see, clearly this is what happened, of course" etc.
 which while good-intentioned, may scare readers off.
 
-We prefer human-friendly language than exact jargon, as long as it's correct,
-even if using general terminology. Example: avoid Git jargon such as _revision_
-or _reference_, preferring the more basic concept _commit_.
+We prefer human-friendly language rather than exact jargon as long as it's
+correct. Example: avoid Git jargon such as _revision_ or _reference_, preferring
+the more basic concept _commit_.
```

Comment:
Out of scope to review the language in this guide but OK...
```suggestion
We prefer general, human-friendly language rather than exact jargon as long as it's
correct. Example: avoid Git jargon such as _revision_ or _reference_, preferring
the more basic terms _commit_ or _version_.
```